dev/plugins/hu.elte.txtuml.validation/src/hu/elte/txtuml/validation/sequencediagram/messages.properties

```diff
@@ -3,3 +3,4 @@ SendExpected_message=Message sending expected.
 InvalidPosition_message=Lifeline position must not be negative.
 InvalidLifelineDeclaration_message=The declared lifeline must be a ModelClass.
 InvalidActionCall_message=Action code is not allowed in this context.
+CycleDetected_message=There is a possible cycle in the SequenceDiagram.
```

Comment:
Insert a newline here please.
Also, i think it should be 'sequence diagram' instead of 'SequenceDiagram', as this sentence is not about the SeqDiag class.